py3status/modules/github.py

```diff
@@ -29,6 +29,12 @@
         '{repo} {issues}/{pull_requests}')*
     format_notifications: Format of `{notification}` status placeholder.
         (default ' N{notifications_count}')
+    github_api_url: Change only if using Enterprise Github to something like
+        https://github.company.com/api/v3.
+        (default 'https://api.github.com')
+    github_url: Change only if using Enterprise Github to somethig like
```

Comment:
`something` typo... and we likely can keep both url configs under two lines each.